atf_core/src/atf_core/atf.py

```diff
@@ -90,23 +90,27 @@ def stop(self, testblock, metric_result = None):
                 raise atf_core.ATFError(error_msg)
 
         else:
-            rospy.loginfo("no user result set for testblock \'%s\'"%testblock)
+            rospy.logwarn("no user result set for testblock \'%s\'"%testblock)
+            metric_result = MetricResult()
+            metric_result.groundtruth.error_message = "!!USER ERROR!!: no user result set for testblock %s"%testblock # TODO use from global field (same as in atf.stop())
```

Comment:
actually, I think we do not need to set the message via a global field
because if you set the result to false, then this us a valid MetricResult which will show up as properfailure during analysis phase...